ecs-init/engine/engine.go

```diff
@@ -32,6 +32,7 @@ const (
 	terminalSuccessAgentExitCode = 0
 	terminalFailureAgentExitCode = 5
 	upgradeAgentExitCode         = 42
+	gpuSupportEnvVar             = "ECS_ENABLE_GPU_SUPPORT"
```

Comment:
I think it would be cleaner to have the config package export env vars as consts rather than scattering them about the code base. Should we move these there?